Assets/Mirage/Runtime/NetworkSceneManager.cs

```diff
@@ -95,7 +95,7 @@ public void Start()
 
             if (Client != null)
             {
-                Client.Authenticated.AddListener(OnClientAuthenticated);
+                Client.Connected.AddListener(OnClientConnected);
```

Comment:
Doesn't this allow SceneMessage to be received before the client is authenticated?

Reply:
Yes, infact I think this should be using `Started` instead of `Connected`

Reply:
Shouldn't client only receive scene messages once he gets authed?

Reply:
this is just adding the handler, the server still sends the message in authenticated.

Reply:
Alright.